### DISCUSSION TITLE
config mirage mods

### DISCUSSION THREAD

**Discussion by kileur2000** [qwbarch/mirage#96] [Q&A] [answered]
in the mirage mods on thunderstore and github you said we can config the mods like the spawn rate of the masked but I can't find it anywhere. The only place I can find is in the setting but the only setting available is the voice of the masked and never delete the recording.

**Comment by qwbarch** [comment, marked as answer]
Sorry for the late response. It's in the ``Mirage.General.cfg`` and ``Mirage.Enemies.cfg`` files.  
Assuming you're using r2modman (or thunderstore), you can access it by clicking on ``Config editor`` and searching up ``Mirage``.
